sale_purchase_lot_inter_company/models/res_company.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2025 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# @author Guillaume MASSON <guillaume.masson@groupevoltaire.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    propagated_serial_number_po_so = fields.Boolean(
+        string="Lots/Serial Numbers are propagated from sales in this company to "
+        "purchase orders"
```

Comment:
I think it is the opposite
```suggestion
        string="Lots/Serial Numbers are propagated from sales to purchase orders"
        "in this company."
```

Reply:
You're totally right !

I changed the string AND the name of the field...
It seems clearer the SN is propagated from the SO to the PO.

But it doesn't reflect which document belongs to the current company, though... any idea ?